### PR TITLE
Specify suite version separately from BWC version

### LINKED ISSUE #37: Can't specify version if installing suites
Two, related problems:
- Can't specify Suite version separately from BWC version
- If you specify a version, it will work for BWC, but will then try to install a suite with the same version. It will also try to pull the suites installer script from the branch matching that version. No way to make this work consistently, if suite versions are going to be different to BWC/ST2 versions.

### PR DESCRIPTION
**Pull request by warrenvw**
Closes #37.

Remaining tasks:
 - [x] Testing

### CODE REVIEW

**Comment by LindsayHill** [comment]
You should combine efforts with @samirsss here #50

**Comment by samirsss** [comment]
@warrenvw can you please verify that you changes work with the changes in #50 as well?

File: scripts/dcfabric-suite-installer-deb.sh [hunk 1]
```diff
@@ -132,35 +135,42 @@ setup_package_cloud_repo() {
...
 }
 
 install_network_essentials_pack() {
   sudo apt-get install -y gcc
   st2 login $USERNAME -p $PASSWORD
+  # FIXME: Use a meaningful VERSION for network_essentials
   if [ "$VERSION" != '' ];
   then
+    echo st2 pack install network_essentials=${VERSION}
     st2 pack install network_essentials=${VERSION}
```
Comment (LindsayHill) [inline scripts/dcfabric-suite-installer-deb.sh]: Will this work? Isn't `$VERSION` here equal to the ST2 version? Network Essentials versioning doesn't match the ST2 versioning. It _might_ match the DC Fabric Suite versioning.

**Comment by samirsss** [comment]
@warrenvw are you planning on updating this PR and merging it. It will be good to have this for the 1.2.0 release of Automation Suites thats coming up soon

**Comment by warrenvw** [comment]
I looked at this issue a couple days ago, but didn't have the time to work on it. I'll need to do some more testing before I merge. I'll provide an estimate shortly as to when I expect this to be complete.

**Pull request closed without merging**